.commonfiles.sha

```diff
@@ -1 +1 @@
-ed099bc2fefd3dbef9319634ab507738608663d4
+0ffcb226039323b7f5b9deb4d34de5193c372537
```

Comment:
not sure where this is used - seems like an odd file to keep as a dot file and remain in the top level directory.  As a followup you may wish to move this file around to something more visible and remove the hidden part (i.e. remove the dot preceediing the file).

Reply:
This is generated by the 
```
$ make updatecommon
```
that downloads the shared shell scripts from istio/common-files, and updates that shasum